CONTRIBUTING.md

```diff
@@ -19,6 +19,10 @@ project documentation. If you cannot find the documentation you're
 looking for, please file a GitHub issue with details of what
 you'd like to see documented.
 
+## Workflow of the App
+
+![Workflow Diagram](docs/img/workflow.jpeg)
```

Comment:
@ashu54-aseem @obulat @zackkrida I here will prefer a markdown rather than just an image. What's your opinion?

Reply:
@Cronus1007 I believe branching arrows and brackets over multiple steps would be a bit jarring and possibly confusing to see in markdown.

Reply:
I think you took the wrong path. I am telling to code in the markdows and while seeing the documentation we can add the Previewed model of the markdown.